README.md

```diff
@@ -28,21 +28,22 @@ New here or looking to install? Check [our website](https://randovania.github.io
 
 <!-- Begin WELCOME -->
 
-Here you will be able to randomize many aspects of either game, while still being ensured it's possible to
-finish without any trick or glitch! What can be randomized?
+Randovania can randomize many aspects of its supported games, all while still ensuring they're completable 
+without any unintended tricks or glitches. Its features include:
 
-* Randomize what can be found in each pickup location, including major upgrades, expansions, keys and artifacts.
+* Randomizing what can be found in each pickup location. Weapons, keys, and more can end up in
```

Comment:
Pickup location or item location? Or just straight up location?

Reply:
Item location is probably better - location isn't strictly true as not every game has "items in every room" like Prime, and "pickup" feels more jargon-y.